code/datums/components/mood.dm

```diff
@@ -257,7 +257,7 @@
 		else
 			if(the_event.timeout)
 				addtimer(CALLBACK(src, .proc/clear_event, null, category), the_event.timeout, TIMER_UNIQUE|TIMER_OVERRIDE)
-			return 0 //Don't have to update the event.
+			return //Don't have to update the event.
```

Comment:
If this is signal it should be returning NONE ?

Reply:
This proc shouldnt be returning anything

Reply:
this proc should be returning NONE

Reply:
this proc should not be returning NONE